src/main/java/org/jabref/model/entry/Month.java

```diff
@@ -93,9 +95,31 @@
             int number = Integer.parseInt(value);
             return Month.getMonthByNumber(number);
         } catch (NumberFormatException e) {
-            return Optional.empty();
+            return parseGermanShortMonth(testString);
         }
     }
+    
+    /**
+	 * Parses a month having the string in German standard form such as
+	 * "Oktober" or in German short form such as "Okt"
+	 * 
+	 * @param value,
+	 *            a String that represents a month in German form
+	 * @return the corresponding month instance, empty if input is not in German
+	 *         form
+         * @author Johannes Preßmar
+	 */
+	private static Optional<Month> parseGermanShortMonth(String value) {
+		String[] shortMonths = new DateFormatSymbols(Locale.GERMAN).getShortMonths();
```

Comment:
Better use the new java 8 Date & Time Api 

Construt a  DateTimeFormatter.ofPattern("MMM", Locale.DE) 
Use YearMonth.parse  and then you can get a Numeric value from that
https://docs.oracle.com/javase/8/docs/api/java/time/YearMonth.html#parse-java.lang.CharSequence-

Reply:
I fixed it. Is it better like this?

Reply:
Yep! Looks better now